examples/sync/plot_sample_rate_test.py

```diff
@@ -0,0 +1,60 @@
+r"""
+======================
+Audio sample rate test
+======================
+
+This example tests the TDT sample rate, which we expect to be ``24414.0625``.
+To test this:
+
+1. Connect the TDT audio output to the sound card input. This can be on
+   the same machine that the TDT is connected to, or a different one.
+   A 1/8" male-male cable running from the headphone monitor to the
```

Comment:
to the...recording computer. Note that you will also need a 1/4" male to 1/8" female adapter to plug it into the RP2 / RP2.1 / RZ6 (whereas the RM1 has an 1/8" jack).